### DISCUSSION TITLE
RFC: `contrast` color

### DISCUSSION THREAD

**Discussion by nandorojo** [tamagui/tamagui#449] [Ideas]
Tamagui's colors/themes currently match the radix 12 scale nicely. One missing piece is buttons that have a solid background color and need a contrasting text color.

For example, if you use a yellow button color, you need black text for contrast. Radix says that `color9` is intended for solid background colors, and lists `white`/`black` as the contrasting text color for each one. I currently don't know of a theme-based way to implement this white/black foreground text. It would be nice to add `$contrastTextColor` or something like that to be `white` or `black`, depending on the corresponding theme.

https://www.radix-ui.com/docs/colors/tests/contrast#aa-large-text

**Comment by natew** [comment]
Haven't checked discussions in a bit, I like the terminology contrast. Will keep this in mind.
